### PR TITLE
fix(rxAge): Fixes parseInt bug

### PR DESCRIPTION
**Pull request by lentusignavus**
JIRA: https://jira.rax.io/browse/FRMW-1000

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] QE LGTM

With really small durations moment returns a small float
parseInt handles this weirdly returning 1 and sometimes 2
Changes implementation to use Math.floor

Issue: https://github.com/rackerlabs/encore-ui/issues/1767

### CODE REVIEW

File: src/utilities/rxAge/scripts/rxAge.js [hunk 1]
```diff
@@ -73,9 +73,9 @@ angular.module('encore.ui.utilities')
         var date = moment(new Date(dateString));
         var diff = now.diff(date);
         var duration = moment.duration(diff);
-        var days = parseInt(duration.asDays(), 10);
-        var hours = parseInt(duration.asHours(), 10);
-        var mins = parseInt(duration.asMinutes(), 10);
+        var days = Math.floor(duration.asDays(), 10);
```
Comment (lentusignavus) [inline src/utilities/rxAge/scripts/rxAge.js]: Removing the `10s`, next commit

**Comment by jonkruse00** [comment]
If I can LGTM I do, when tests pass

**Comment by 100stacks** [comment]
@lentusignavus it looks you will have to update `package.json` https://github.com/rackerlabs/encore-ui/blob/1.x-LTS/package.json#L16

from:
```js
"postinstall": "grunt hooks",
```

to:
```js
"postinstall": "cd utils/rx-page-objects; npm i; cd -; grunt hooks",
```

...so that `npm` installs the `rx-page-objects` node modules.

You also need to add https://github.com/rackerlabs/encore-ui/blob/master/utils/rx-page-objects/protractor.conf.js to back port `1.x-LTS` to run the midways.  @Droogans anything else?

**Comment by 100stacks** [comment]
@lentusignavus you can delete this line https://github.com/rackerlabs/encore-ui/blob/master/utils/rx-page-objects/protractor.conf.js#L1 when you add the file, since we are now using `ESLint`. 👍

**Comment by Droogans** [comment]
It appears some commits from master never made it onto the 1.x-LTS branch for maintaining the testing infrastructure. I'll see what I can do real fast.

**Comment by Droogans** [comment]
I'll play https://github.com/rackerlabs/encore-ui/pull/1742 onto the 1.x-LTS branch via this pull request to streamline the review process. You will have to ignore those comments here though!

**Comment by Droogans** [comment]
Ah, still got problems....

**Comment by 100stacks** [comment]
@Droogans, thanks for the assist...on a late Friday at that! 😄 💯

**Comment by lentusignavus** [comment]
@100stacks addressing comments now

**Comment by Droogans** [comment]
@lentusignavus @100stacks I forgot one more commit to skip alerts in chrome. Just a moment.

**Comment by Droogans** [comment]
Don't forget that the visreg system is going to be submitting a very large screenshots PR since it will detect a new baseline branch `1.x-LTS`.

This is just a baseline image set for all successive PRs against the `1.x-LTS` branch in the future. Go ahead and merge it.

**Comment by Droogans** [comment]
Wow, this rabbit hole just keeps getting deeper 😆.

@CITguy can you cherry pick in the rxExample fixes you did? They're making the javascript error detection plugin in protractor freak out with all the 404's that didn't get backported to the demo site for the 1.x branch!

**Comment by lentusignavus** [comment]
gonna squash before merge to update the commit msg

**Comment by Droogans** [comment]
@lentusignavus Please make sure you only squash the rxAge commits on top of the pre-existing ones that I've cherry picked in. You can do this with an interactive rebase.

They were isolated before this branch, they should remain isolated in the `1.x-LTS` branch as well.

File: src/utilities/rxAge/scripts/rxAge.js [hunk 2]
```diff
@@ -73,9 +73,9 @@ angular.module('encore.ui.utilities')
         var date = moment(new Date(dateString));
         var diff = now.diff(date);
         var duration = moment.duration(diff);
-        var days = parseInt(duration.asDays(), 10);
-        var hours = parseInt(duration.asHours(), 10);
-        var mins = parseInt(duration.asMinutes(), 10);
+        var days = Math.floor(duration.asDays());
+        var hours = Math.floor(duration.asHours());
+        var mins = Math.floor(duration.asMinutes());
```
Comment (CITguy) [inline src/utilities/rxAge/scripts/rxAge.js]: **NOTE**: This is the functional fix. All else is to get CI services working correctly.

**Comment by lentusignavus** [comment]
Rebased, maintaining @Droogans history

**Comment by CITguy** [comment]
LGTM

**Comment by CITguy** [comment]
@Droogans: I've cherry-picked the rxExample performance update. Hopefully CI services start working again.

**Pull request merged**